site/content/en/docs/drivers/_index.md

```diff
@@ -31,6 +31,14 @@ To do so, we use the [Docker Machine](https://github.com/docker/machine) library
 * [VMware Fusion]({{<ref "vmware.md">}}) - VM
 * [SSH]({{<ref "ssh.md">}}) - remote ssh
 
+### macOS M1 (Silicon)
+
+The following drivers support macOS M1 users:
```

Comment:
In general, it would be better if this talked about CPU architectures (arm64) rather than CPU _models_.

There is nothing magic about Apple M1 (that wouldn't apply to M2), and the issue is the same on Linux.

I'm not sure if podman on mac works at all for minikube, but I guess it is "experimental" after all.

The CoreOS VM image is available as aarch64 (and x86_64), so it should be possible to test it at least.